.vscode/launch.json

```diff
@@ -0,0 +1,39 @@
+{
```

Comment:
Does it make sense to keep a .vscode folder in there?
Maybe add to the .gitignore